tools/run-wpt.mjs

```diff
@@ -90,87 +103,98 @@ if (HEADLESS === 'true') {
   log('Running WPT in headful mode...');
 }
 
-const wptRunArgs = [
```

Comment:
This change block looks scary, but in fact I only moved running logic to `if (RUN_TESTS === 'true')`.